### DISCUSSION TITLE
Change displayed name of Squeezbox Radio

### DISCUSSION THREAD

**Discussion by SvenSchiffner** [music-assistant/support#2635] [Feature requests and ideas]
I have multiple Squeezebox Radios in my home. These are identified by MASS as controller with a hexadecimal number (maybe the MAC). In MASS itself and HA I can change the name easily. But in the Radio itself it still display the "controller <hexadecimal number>". It would be very nice (mainly cosmetic) to change that name.
Maybe this is also problem with squeezelite players, but I never used them.

**Comment by OzGav** [comment]
Going through old posts. This is supserseded. You can edit the friendly name for any player now.
